### ISSUE TITLE
settings for multi-gpu training

### ISSUE THREAD

**Issue by hz0567** [mapo-t2i/mapo#8]
Hi, thanks for sharing your code. I am trying to reproduce your model with multi-gpu training on the pick-pic v2 dataset. I am wondering  how did you set the gradient accumulation steps for multi-gpu (8 H100) training.

**Comment by jiwooya1000** [comment]
Hello @HongzhengYang,

We set the per-device train batch size to 32 and gradient accumulation steps to 8, eventually having a total batch size of 2048 with DeepSpeed 2 for 8 H100.

**Comment by hz0567** [comment]
Thanks for your quick response.
